content/docs/render-props.md

```diff
@@ -299,27 +301,30 @@ class MouseTracker extends React.Component {
 }
 ```
 
-In this example, each time `<MouseTracker>` renders, it generates a new function as the value of the `<Mouse render>` prop, thus negating the effect of `<Mouse>` extending `React.PureComponent` in the first place!
+Dalam contoh ini, setiap kali `<MouseTracker>` di-_render_, ia akan menghasilkan sebuah fungsi baru sebagai nilai dari props `<Mouse render>`, sehingga menghilangkan efek `<Mouse>` yang meng-_extend_ `React.PureComponent` sejak awal!
 
-To get around this problem, you can sometimes define the prop as an instance method, like so:
+Untu mengatasi permasalahan ini, Anda sekali waktu dapat mendefinisikan props ini sebagai _instance_ dari sebuah metode, seperti contoh berikut:
 
 ```js
 class MouseTracker extends React.Component {
   // Defined as an instance method, `this.renderTheCat` always
   // refers to *same* function when we use it in render
+  // Didefinisikan sebagai *instance* dari metode, `this.renderTheCat` selalu
+  // merujuk kepada fungsi yang *sama* ketika kita menggunakannya di render
   renderTheCat(mouse) {
     return <Cat mouse={mouse} />;
   }
 
   render() {
     return (
       <div>
-        <h1>Move the mouse around!</h1>
+        <h1>Gerak-gerakkan tetikus!</h1>
```

Comment:
The translation was not consistent.

Reply:
My bad.. I think I'll go with `gerak-gerakkan tetikus`